modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-creation-and-sharing_{context}"]
+= Configuring a VPC to share within your AWS organization
+
+You can share your configured VPC with another AWS user account if that account is within your current AWS organization.
+
+.Procedure
+
+. On your VPC-owning AWS account ID, you must create your VPC to your desired specifications in the link:https://us-east-1.console.aws.amazon.com/vpc/[VPC section of the AWS console]. 
```

Comment:
This sentence confuses me. If the AWS account already owns the VPC (per the first phrase), why do you need to create the VPC?